bin/gagarin

```diff
@@ -36,6 +36,7 @@ program
   .option('-m, --mute-build', 'do not show build logs', false)
   .option('-f, --flavor <name>', 'default flavor of api (promise, fiber)', 'promise')
   .option('-T, --startup-timeout <ms>' ,'server startup timeout [5000]', intParse, 5000)
+  .option('-U, --meteor-startup-timeout <ms>' ,'initial meteor startup timeout [120000]', intParse, 120000);
```

Comment:
@marekrogala I am wondering why do we need another option here. Can't we just use `startupTimeout`? If the reason is that builds would usually take a little more time to complete then why not use `buildTimeout` instead? I guess it would be a lot more intuitive.